app/views.py

```diff
@@ -134,6 +131,19 @@ def index(request, *args, **kwargs):
         'num_institutions': num_institutions,
         'num_countries': num_countries,
         'num_cities': num_cities,
+
+    }
+    # extra_context=generate_context( **kwargs)
+    # context.update(extra_context)
+    return render(request, 'index.html', context)
+
+def generate_context( **kwargs):
+    scientists, num_scientists, num_institutions, num_countries, num_male_scientists, num_female_scientists, \
```

Comment:
Hay variables que se definen pero nunca se utilizan dentro del scope de la función y por lo tanto son redundantes

Reply:
Exacto, las funcion **__get_data_map** retorna de esa forma los datos
![abc](https://user-images.githubusercontent.com/12649261/121826156-58a6ae00-cc84-11eb-9d1c-b80d0b2f57ba.png)

Reply:
Se puede usar underscore para variables que son devueltas por `get_data_map` pero que no se utilizan en la función. Fijate aquí https://medium.com/python-features/naming-conventions-with-underscores-in-python-791251ac7097

Reply:
Done

Reply:
https://github.com/joausaga/cipe/pull/19/commits/6a5b3007ad8b38fc8bdb576c2e1c6fcc33a9952a